_tags

```diff
@@ -0,0 +1,40 @@
+true : bin_annot, safe_string, package(bytes)
```

Comment:
Regarding `debug` feed back about https://github.com/dbuenzli/topkg/issues/54 is welcome.